examples/vibrator/main.py

```diff
@@ -1,11 +1,7 @@
-
+"""Plyer Vibrator Example."""
```

Comment:
Simple, one-line docstrings like this don't add any information and should be avoided. (We know this is a Plyer Vibrator Example, because it's examples/vibrator/main.py in Plyer. Similarly we know that VibrationApp is the Main Application, and that build() returns the root widget, etc., without needing docstrings.) Also, remember to follow our standards (look at the rest of the code). We use `''''''` for docstrings, not `""""""`.